ceramic.js

```diff
@@ -10,43 +10,43 @@ import {ceramicNodeUrl, metaverseProfileDefinition} from './constants.js';
 
 const login = async () => {
   const address = await getMainnetAddress();
-  const threeIdConnect = new ThreeIdConnect();
-  const authProvider = new EthereumAuthProvider(window.ethereum, address);
-  await threeIdConnect.connect(authProvider);
-  const provider = await threeIdConnect.getDidProvider();
+  // const threeIdConnect = new ThreeIdConnect();
+  // const authProvider = new EthereumAuthProvider(window.ethereum, address);
+  // await threeIdConnect.connect(authProvider);
+  // const provider = await threeIdConnect.getDidProvider();

```

Comment:
Are we commenting this code out because we are re-adding DID/Ceramic back soon? Or will we not need these commented out lines and can delete them?

Reply:
We might. Avaer suggested to comment them.